app/src/main/java/woowacourse/movie/domain/MovieSchedule.kt

```diff
@@ -0,0 +1,59 @@
+package woowacourse.movie.domain
+
+import java.time.DayOfWeek
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+class MovieSchedule(
+    private val startDate: LocalDate,
+    private val endDate: LocalDate,
+) {
+    private val dateFormatter = DateTimeFormatter.ofPattern(DATE_PATTERN)
+    private var selectedDate = ""
+    private var selectedTime = ""
+
+    fun setDate(date: String) {
+        selectedDate = date
+    }
+
+    fun setTime(time: String) {
+        selectedTime = time
+    }
+
+    fun getDate(): String = selectedDate
+
+    fun getTime(): String = selectedTime
+
```

Comment:
이 부분은 아래 코드와 어떻게 다른가요? 제 눈에 똑같아 보여서요!
```suggestion
    var selectedDate = ""
    var selectedTime = ""
```